File: 1078. Occurrences After Bigram/main.py
class Solution(object):
    def findOcurrences(self, text, first, second):
        """
        :type text: str
        :type first: str
        :type second: str
        :rtype: List[str]
        """
        text = text.split()
        i=0
        result = []
        while i<len(text)-2:
            if text[i] == first and text[i+1] == second:
                result.append(text[i+2])
                i+=2
            else:
                i+=1
        return result

File: 1078. Occurrences After Bigram/main.java
class Solution {
    public String[] findOcurrences(String text, String first, String second) {
        String[] str_list = text.split(" ");
        int i=0;
        ArrayList<String> result = new ArrayList<String>();
        while (i<str_list.length-2){
            if (str_list[i].equals(first) && str_list[i+1].equals(second)){
                result.add(str_list[i+2]);
                i+=2;
            } else {
                i+=1;
            }
            
        }
        return result.toArray(new String[result.size()]);
    }
}